docs-partials/builder/qemu/Config-not-required.mdx

```diff
@@ -300,6 +300,11 @@
   to qemu, allowing it to choose the default. This may be needed when running
   under macOS, and getting errors about sdl not being available.
 
+- `vga` (string) - The type of VGA card to emulate
```

Comment:
I'm not sure why GitHub is showing this in read. But it might have something to do with the ``` backticks for the code block not having a blank line after. I'm just calling it out because I see it but I wouldn't spend much cycle on figuring it out.

Reply:
Look like the parser's lost it at the ```json segment, I've seen that on other project too.